src/_P042_Candle.ino

```diff
@@ -76,422 +83,440 @@ enum ColorType {
-#define PLUGIN_VALUENAME3_042 "Type"
+# define PLUGIN_042
+# define PLUGIN_ID_042         42
+# define PLUGIN_NAME_042       "Output - NeoPixel (Candle)"
+# define PLUGIN_VALUENAME1_042 "Color"
+# define PLUGIN_VALUENAME2_042 "Brightness"
+# define PLUGIN_VALUENAME3_042 "Type"
 
 boolean Plugin_042(uint8_t function, struct EventStruct *event, String& string)
 {
   boolean success = false;
 
   switch (function)
   {
-
     case PLUGIN_DEVICE_ADD:
-      {
-        Device[++deviceCount].Number = PLUGIN_ID_042;
-        Device[deviceCount].Type = DEVICE_TYPE_SINGLE;
-        Device[deviceCount].VType = Sensor_VType::SENSOR_TYPE_TRIPLE;
-        Device[deviceCount].Ports = 0;
-        Device[deviceCount].PullUpOption = false;
-        Device[deviceCount].InverseLogicOption = false;
-        Device[deviceCount].FormulaOption = false;
-        Device[deviceCount].ValueCount = 3;
-        Device[deviceCount].SendDataOption = true;
-        Device[deviceCount].TimerOption = true;
-        Device[deviceCount].GlobalSyncOption = false;
-        break;
-      }
+    {
+      Device[++deviceCount].Number           = PLUGIN_ID_042;
+      Device[deviceCount].Type               = DEVICE_TYPE_SINGLE;
+      Device[deviceCount].VType              = Sensor_VType::SENSOR_TYPE_TRIPLE;
+      Device[deviceCount].Ports              = 0;
+      Device[deviceCount].PullUpOption       = false;
+      Device[deviceCount].InverseLogicOption = false;
+      Device[deviceCount].FormulaOption      = false;
+      Device[deviceCount].ValueCount         = 3;
+      Device[deviceCount].SendDataOption     = true;
+      Device[deviceCount].TimerOption        = true;
+      Device[deviceCount].GlobalSyncOption   = false;
+      break;
+    }
 
     case PLUGIN_GET_DEVICENAME:
-      {
-        string = F(PLUGIN_NAME_042);
-        break;
-      }
+    {
+      string = F(PLUGIN_NAME_042);
+      break;
+    }
 
     case PLUGIN_GET_DEVICEVALUENAMES:
-      {
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_042));
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], PSTR(PLUGIN_VALUENAME2_042));
-        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[2], PSTR(PLUGIN_VALUENAME3_042));
-        break;
-      }
+    {
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_042));
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], PSTR(PLUGIN_VALUENAME2_042));
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[2], PSTR(PLUGIN_VALUENAME3_042));
+      break;
+    }
 
     case PLUGIN_GET_DEVICEGPIONAMES:
-      {
-        event->String1 = formatGpioName_output(F("Data"));
-        break;
-      }
-      
+    {
+      event->String1 = formatGpioName_output(F("Data"));
+      break;
+    }
+
     case PLUGIN_WEBFORM_LOAD:
-      {
-        addHtml(F("<script src=\"jscolor.min.js\"></script>\n"));
+    {
+      addHtml(F("<script src=\"jscolor.min.js\"></script>\n"));
```

Comment:
I think there are even `<script ...` functions in somewhere around where other `html_XXX` functions are.